src/runtime/object.h

```diff
@@ -340,7 +340,8 @@ inline obj_res nat_lxor(b_obj_arg a1, b_obj_arg a2) { return lean_nat_lxor(a1, a
 
 // =======================================
 // Integers
-object * mk_int_obj_core(mpz const & m);
+obj_res mpz_to_int_core(mpz const & m);
+inline obj_res mk_int_obj_core(mpz const & m) { return mpz_to_int_core(m); }
```

Comment:
The function `mk_int_obj_core` did not exist, but `mpz_to_int_core` does. I just copied the pattern used in `mk_nat_obj_core` here.